### DISCUSSION TITLE
Slideshow controls

### DISCUSSION THREAD

**Discussion by hhanzo1** [LycheeOrg/Lychee#2552] [Q&A] [answered]
Apologies if this has been asked already but I couldnt find the info.

I checked for keyboard shortcuts for slideshow but there wasnt any.

Is their a slideshow feature and show does one control it?

**edit:** actually I found the slide show icon (top right) and adjusted the refresh rate from 30 to 3. The slide show photos appear to be randomly chosen.  

Is it possible to start a slide show from first photo in the album?

Thanks.

**Comment by ildyria** [comment]
Are you talking about the `/frame` mod ?
Because that one is just showing pictures at random, so previous and next are not very much helpful.
We do not have a slideshow module implemented which would trigger the next picture after x seconds (though I don't think it would be to hard to add with vuejs).

I can see to add your suggestion to v6 a bit later. My priority is getting a beta out first (after fixing a few more things).

**Reply by ildyria** [reply, marked as answer]
**Done in version 6.**

**Comment by hhanzo1** [comment]
Yes, I think so.  It's the little computer monitor looking icon on the top right.

OK. Thanks. No worries. Not urgent.

Looking forward to the v6 beta. :-)
